### PR TITLE
Update User Audit Scripts for Pagination

### PR DESCRIPTION
**Pull request by ababhis2**
VM-5208

The script will work for both paginated and unpaginated API.

paginated response for useraudit.py
image

for audit script
image

### CODE REVIEW

File: User Audit/useraudit.py [hunk 1]
```diff
@@ -31,27 +30,42 @@ def flatten(x, name='', exclude=exclude):
     return out
 
 token = sys.argv[1]
-base_url = "https://api.kennasecurity.com"
-users_url= base_url + "/users"
+# increase per_page to get more data per request
+per_page = 10
```
Comment (hingandh) [inline User Audit/useraudit.py:34]: can this be configurable like user can adjust this number?

**Review by jaredkalmus — APPROVED** [review]
Performed a test run for my environment and all users were returned with appropriate data

File: User Audit/useraudit.py (hunk 1 above)
Comment (ababhis2) [reply User Audit/useraudit.py]: these scripts are sent to the client so either CX or client can customize the scripts. I have added a small README snippet about `per_page` but I do not see the need to make it a CLI option. I do not like the way CLI is being used in user audit script but not planning to change it

**Review by hingandh — APPROVED** [review]

File: User Audit/README.md [hunk 2]
```diff
@@ -33,5 +33,8 @@ You can set an environment variable in Windows using the setx command in the com
 In macOS or Linux, you can set an environment variable in the terminal using the export command:
 *export API_KEY=your-api-key*
 
+### 4. Pagination Size
```
Comment (macesped) [inline User Audit/README.md:36]: Nip: this should go after # 3. Also, it should keep the same format: `4:`.

**Pull request merged**